deploy/examples/monitoring/service-monitor.yaml

```diff
@@ -13,7 +13,6 @@ spec:
     matchLabels:
       app: rook-ceph-mgr
       rook_cluster: rook-ceph
-      ceph_daemon_id: a
```

Comment:
do we need the label "mgr_role: active" here?

Reply:
Good catch! I think we do need the new `mgr_role` here otherwise we will select both mgrs (standby also). But standby normally doesn't return `/metric` EP. I updated the yaml to include the new label.